src/main/java/com/solvd/carina/demo/mobile/saucedemomobile/android/CheckOutYourInformationScreen.java

```diff
@@ -0,0 +1,57 @@
+package com.solvd.carina.demo.mobile.saucedemomobile.android;
+
+import com.solvd.carina.demo.mobile.saucedemomobile.common.CheckOutYourInformationScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.OverviewScreenBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = CheckOutYourInformationScreenBase.class)
+public class CheckOutYourInformationScreen extends CheckOutYourInformationScreenBase {
+
+    @FindBy(xpath = "//android.widget.TextView[@text='CHECKOUT: INFORMATION']")
+    private ExtendedWebElement screenTitle;
+
+    @FindBy(xpath = "//android.widget.EditText[@content-desc=\"test-First Name\"]")
+    private ExtendedWebElement firstNameField;
+
+    @FindBy(xpath = "//android.widget.EditText[@content-desc=\"test-Last Name\"]")
+    private ExtendedWebElement lastNameField;
+
+    @FindBy(xpath = "//android.widget.EditText[@content-desc=\"test-Zip/Postal Code\"]")
+    private ExtendedWebElement zipCodeField;
+
+    @FindBy(xpath = "//android.view.ViewGroup[@content-desc=\"test-CONTINUE\"]/android.widget.TextView")
```

Comment:
```suggestion
    @FindBy(xpath = "//android.view.ViewGroup[@content-desc=\"test-CONTINUE\"]")
```
please check if this locator works